websites/M/movie-web/presence.ts

```diff
@@ -0,0 +1,294 @@
+const presence = new Presence({
+		clientId: "1120627624377589820",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+interface MovieMedia {
+	meta: {
+		meta: {
+			title: string;
+			id: string;
+			year: string;
+			poster: string;
+			type: "movie";
+		};
+		imdbId: string;
+		tmdbId: string;
+	};
+	captions: {
+		langIso: string;
+		url: string;
+		type: string;
+	}[];
+	episode?: undefined;
+}
+
+interface SeriesMedia {
+	meta: {
+		meta: {
+			title: string;
+			id: string;
+			year: string;
+			poster: string;
+			type: "series";
+			seasons: {
+				id: string;
+				number: number;
+				title: string;
+			}[];
+			seasonData: {
+				id: string;
+				number: number;
+				title: string;
+				episodes: {
+					id: string;
+					number: number;
+					title: string;
+				}[];
+			};
+		};
+		imdbId: string;
+		tmdbId: string;
+	};
+	episode: {
+		episodeId: string;
+		seasonId: string;
+	};
+	captions: {
+		langIso: string;
+		url: string;
+		type: string;
+	}[];
+}
+
+interface DevMetaData {
+	media: MovieMedia | SeriesMedia;
+	state: {
+		progress: {
+			time: number;
+			duration: number;
+		};
+		mediaPlaying: {
+			hasPlayedOnce: boolean;
+			isDragSeeking: boolean;
+			isFirstLoading: boolean;
+			isLoading: boolean;
+			isPaused: boolean;
+			isPlaying: boolean;
+			isSeeking: boolean;
+			playbackSpeed: number;
+			volume: number;
+		};
+	};
+}
+
+interface ProdMetaData {
+	meta:
+		| {
+				meta: {
+					title: string;
+					id: string;
+					year: string;
+					poster: string;
+					type: "movie";
+				};
+				imdbId: string;
+				tmdbId: string;
+		  }
+		| {
+				meta: {
+					title: string;
+					id: string;
+					year: string;
+					poster: string;
+					type: "series";
+					seasons: {
+						id: string;
+						number: number;
+						title: string;
+					}[];
+					seasonData: {
+						id: string;
+						number: number;
+						title: string;
+						episodes: {
+							id: string;
+							number: number;
+							title: string;
+						}[];
+					};
+				};
+				imdbId: string;
+				tmdbId: string;
+		  };
+	episode?: {
+		episodeId: string;
+		seasonId: string;
+	};
+	progress: {
+		time: number;
+		duration: number;
+	};
+}
+
+presence.on("UpdateData", async () => {
+	const { pathname, href, host } = window.location,
```

Comment:
We prefer documen.location
```suggestion
	const { pathname, href, host } = document.location,
```